src/test/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAuthenticationManagerTest.java

```diff
@@ -151,11 +151,12 @@ public void calculateUpdatedHMAC_PartialLogin() {
     @Test
     public void destroyUserSession() throws JsonProcessingException, SegueDatabaseException, NoUserLoggedInException {
         RegisteredUser mockUser = createNiceMock(RegisteredUser.class);
-        expect(mockUser.getSessionToken()).andReturn(1);
+        expect(mockUser.getSessionToken()).andReturn(1).times(2);
```

Comment:
why twice?

Reply:
Because the validation performs two comparisons on that property. Would it be better to extract that to a variable before the if statement rather than accessing it directly?
`if (userFromDatabase.getSessionToken() == -1 || !userFromDatabase.getSessionToken().toString().equals(userSessionToken))`

Reply:
I'm not sure why this was needed here though... 🤔 
Pretty sure the validation checks were updated in another ticket but the tests must have been fine there because the checks passed...